gensim/models/ldamodel.py

```diff
@@ -698,7 +698,9 @@ def rho():
             dirty = False
 
             reallen = 0
-            for chunk_no, chunk in enumerate(utils.grouper(corpus, chunksize, as_numpy=chunks_as_numpy)):
+            for chunk_no, chunk in enumerate(utils.grouper(
+                                                       corpus, chunksize, as_numpy=chunks_as_numpy,
```

Comment:
Bad indent. Also, line too long -- best split this long expression into two smaller expressions (first assign `utils.grouper`, then `enumerate`).

Reply:
still not changed